src/app/map/[id]/Controls/parent-hierarchy.static.tsx

```diff
@@ -0,0 +1,46 @@
+import { ChevronDown } from "lucide-react";
+import type { HexTileData } from "../State/types";
+import { HierarchyTile } from "../Tile/hierarchy-tile.static";
+import {
+  _getParentHierarchy,
+  _shouldShowHierarchy,
+} from "../State/hierarchy.utils";
+
+interface ParentHierarchyProps {
+  centerCoordId: string;
+  items: Record<string, HexTileData>;
+}
+
+export const ParentHierarchy = ({
+  centerCoordId,
+  items,
+}: ParentHierarchyProps) => {
+  const hierarchy = _getParentHierarchy(centerCoordId, items);
+
+  if (!_shouldShowHierarchy(hierarchy)) {
+    return null;
+  }
+
+  return (
+    <div className="fixed right-4 top-1/2 z-30 -translate-y-1/2">
+      <div className="flex flex-col items-center gap-2 rounded-lg bg-transparent px-3 py-4">
+        {hierarchy.map((item, index) => (
+          <div
+            key={item.metadata.coordId}
```

Comment:
I'm not sure if the key has to be unique for the whole html page or not. If it has just to be unique among these hierarchy elements keep it like this but else just add a "hierarchy-" prefix.